src/main/scala/org/constellation/consensus/HTTPNodeRemoteSender.scala

```diff
@@ -0,0 +1,32 @@
+package org.constellation.consensus
+import org.constellation.consensus.NodeRemoteSender.TransactionsProposal
+import org.constellation.primitives.PeerData
+
+import scala.concurrent.Future
+
+class HTTPNodeRemoteSender extends NodeRemoteSender {
+
+  val blockProposalRoute = "/block-round/proposal"
```

Comment:
I have thought about whether string constants such as this should maybe come from a central configuraton?

Reply:
@Nikolaj-K thanks for comment. Timeouts and others will go to config file and can overwritten during app startup or environmental variable of particular node, however this one here is api path and can't be changed without breaking backward compatibility, the second reason to keep it here is that you want immediately know where the api calls are going.

Reply:
@wkoszycki Right. Whereas less substantial numbers like timeouts (that are changed frequently) should be in a config, this sort of sensitive constant string should not be at the same config, because then users would be mislead to think they can also change them. It's not a configuration.

Whether it makes it easier to find "where the api calls are going" if those strings are members of one of the many node classes is another question. 
Here at least it's located at the beginning of the file, but my thinking was that such information could still be traced back to a more central place. This doesn't have to be intermingled with more dynamical config values. 
Anyway, that was just my first thought here, I must reverse engineer the API for myself to understand the broader picture first :v: